github/scss/common-blocks/sign-up/header/header-navigation/header-navigation.scss

```diff
@@ -12,9 +12,17 @@
   margin-left: 1vw;
 }
 
+.header-navigation__link:hover {
```

Comment:
Better way for avoiding copy 

`header-navidation {
      &__link:hover {
      }
      &__dropwon-item {
      
      }
}`